src/util/constants.js

```diff
@@ -19,17 +9,25 @@ export const defaultExerciseSites = [
     "subdomain": "web",
     "name": "Hellotalk"
   },
+  {
+    "hostname": "www.zeeguu.org",
+    "href": "https://www.zeeguu.org/practice/#!/practice/plan/0",
+    "pathname": "//practice/#!/practice/plan/0",
+    "regex": "*://*.www.zeeguu.org/*",
+    "tld": "org",
+    "domain": "zeeguu",
+    "subdomain": "www",
+    "name": "Zeeguu"
+  }
   // 'https://www.brainscape.com/'
 ];
 
 export const defaultExerciseSite = defaultExerciseSites[0];
 
-export const defaultexerciseDuration = 5; // minutes
-
-export const exerciseDuration = 5 * 60 * 1000; // 5 minutes, in ms
+export const defaultexerciseDuration = 5 * 60 * 1000; // 5 minutes
 
 export const defaultTimeout = 5 * 60 * 1000; // 5 minutes
```

Comment:
this is too much - 30 seconds or one minute would already be better.

Reply:
This PR did not change any of the time constants-